browser/ui/views/frame/brave_tab_strip_region_container.h

```diff
@@ -0,0 +1,51 @@
+/* Copyright (c) 2022 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVE_BROWSER_UI_VIEWS_FRAME_BRAVE_TAB_STRIP_REGION_CONTAINER_H_
+#define BRAVE_BROWSER_UI_VIEWS_FRAME_BRAVE_TAB_STRIP_REGION_CONTAINER_H_
+
+#include "chrome/browser/ui/views/frame/tab_strip_region_view.h"
+
+namespace views {
+class ScrollView;
+}
+
+// Wraps TabStripRegion and show it vertically.
+class BraveTabStripRegionContainer : public views::View {
```

Comment:
We shouldn't prepend `Brave` to something that is specific to vertical tab strip. These classes should be named based on vertical tab strip.